### ISSUE TITLE
Docs don't reference ACS url when configuring SP in IdP-first auth

### ISSUE THREAD

**Issue by brandon-leapyear** [simplesamlphp/simplesamlphp#1164]
I was trying to build an SP to handle a SAML response from an external, already running IdP. I thought I had everything set up, but every time the IdP redirected to the PHP file I wrote containing `$auth->requireAuth()`, it was never authenticating the SAML response.

After a LOT of digging, I found the [IdP Quickstart](https://simplesamlphp.org/docs/stable/simplesamlphp-idp#section_7), which shows the URL that the IdP should send the response to: 
```
<?php
$metadata['https://sp.example.org/simplesaml/module.php/saml/sp/metadata.php/default-sp'] = [
    'AssertionConsumerService' => 'https://sp.example.org/simplesaml/module.php/saml/sp/saml2-acs.php/default-sp',
    'SingleLogoutService'      => 'https://sp.example.org/simplesaml/module.php/saml/sp/saml2-logout.php/default-sp',
];
```
I then figured that I had to set the ACS Location of the SP to `/simplesaml/module.php/saml/sp/saml2-acs.php/name-of-my-sp`, which worked.

My recommendation is to add a section in the [SP docs](https://simplesamlphp.org/docs/stable/simplesamlphp-sp) containing docs/examples for IdP-initiated flows where the IdP is not controlled by you.

**Comment by tvdijen** [comment]
Why didn't you configure the IdP using your SP's metadata?

**Comment by brandon-leapyear** [comment]
Oh I see, it's set in the generated metadata. Ok, maybe this issue can be closed.

I'm new to SAML, and I'm in charge of implementing SAML authentication for our software. I basically had an example SAML document from the customer and was told to consume that in our software, so the first step I took was to implement an IdP that returned a SAML response in the same format as that example SAML doc. So I created a Docker container with the IdP and hardcoded the SP metadata so that the SAML document came back as I wanted. And then I implemented the SP and filled in `authsources.php` based on the SP metadata I had already hardcoded.

I understand now that this workflow was kind of backwards, but I guess the requirements that were given to me forced me to work backwards compared to a "typical" SAML implementation
